rs/nervous_system/integration_tests/src/pocket_ic_helpers.rs

```diff
@@ -1651,6 +1654,20 @@ pub mod sns {
             };
             Decode!(&result, sns_pb::GetUpgradeJournalResponse).unwrap()
         }
+
+        pub async fn advance_target_version(
+            pocket_ic: &PocketIc,
+            sns_governance_canister_id: PrincipalId,
+            target_version: Version,
+        ) -> AdvanceTargetVersionResponse {
+            let payload = AdvanceTargetVersionRequest {
+                target_version: Some(target_version),
+            };
+            pocket_ic
+                .call(sns_governance_canister_id, payload)
+                .await
+                .unwrap()
```

Comment:
I implemented this with the CallCanisters trait as it's less verbose and does the same thing. However, for consistency, I'd be happy to change it to the way we typically call canisters in the pocketic integration tests.